src/test/java/uk/ac/ebi/subs/biostudies/client/BioStudiesClientTest.java

```diff
@@ -62,19 +68,19 @@ public void loginFailure() {
         BioStudiesSession session = client.initialiseSession();
 
         //don't expect to get here
-        Assert.assertNotNull(session);
+        assertNotNull(session);
```

Comment:
I think we don't need this line. We are expecting an exception in this test. If we won't get it, than the test will fail anyway.